content/docs/command-reference/repro.md

```diff
@@ -48,7 +48,9 @@ pipeline(s). Then it executes the corresponding commands (`cmd` field of
 
 There are a few ways to restrict what will be regenerated by this command: by
 specifying specific reproduction [`targets`](#options), or by using certain
-command [options](#options), such as `--single-item` or `--all-pipelines`.
+command [options](#options), such as `--single-item` or `--all-pipelines`. It is
+also possible to use `--glob` and `--glob-stages` to match targets with
+[glob](<https://en.wikipedia.org/wiki/Glob_(programming)>) patterns.
```

Comment:
In https://github.com/iterative/dvc.org/pull/2362#issuecomment-815175098 I meant to mention that another way is to specify stage outputs (which will cause the whole stage to be reproed (right?)).

No need to change general info. about `--glob` in this PR (only further down to update it as per this PR's actual topic: stage outputs as `targets`.)

Reply:
I need a bit of clarification here. If we don't want information about `*glob*` flags here (thank you for pointing it out) I don't see what should be added to this paragraph, because I understand that from now on generic target could be also any defined stage output (and it is described in section about targets, to which `target` word links). So imo this part should be left as it was at the beginning (without any changes that I've added). Are we on the same page about that?

Reply:
We were on the same page, yes.